src/index.js

```diff
@@ -36,6 +36,8 @@ class OnVisible extends Component {
             this.setState({
                 visible: true,
                 top
+            }, function() {
+                if (this.props.onChange) this.props.onChange(this.state.visible);
```

Comment:
actually, this function isn't bound too, so needs to be a fat arrow to have access to `this`.